stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/utils/extensions/Message.kt

```diff
@@ -51,4 +29,7 @@ internal fun Message.hasNoAttachments(): Boolean = attachments.isEmpty()
 
 internal fun Message.isEphemeral(): Boolean = type == ModelType.message_ephemeral
 
-internal fun Message.getCreatedDate(): Date? = createdAt ?: createdLocallyAt
+internal fun Message.getCreationDate(): Date =
+    createdAt ?: createdLocallyAt ?: throw IllegalStateException("Message $id : \"${text}\" missing creation metadata")
```

Comment:
We could use this in a lot of places within our codebase, though in some places it would have to be nullable. Maybe we should add a `getCreationDateOrNull` variant.